### ISSUE TITLE
KeyError: '3.7.5' when importing uncompyle6

### ISSUE THREAD

**Issue by laike9m** [rocky/python-xdis#52]
Platform: Windows 10, Python 3.7.5

```python
>>> import uncompyle6
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\uncompyle6\__init__.py", line 48, in <module>
    import uncompyle6.semantics.pysource
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\uncompyle6\semantics\pysource.py", line 136, in <module>
    from uncompyle6.parsers.treenode import SyntaxTree
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\uncompyle6\parsers\treenode.py", line 3, in <module>
    from uncompyle6.scanners.tok import NoneToken
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\uncompyle6\scanners\tok.py", line 163, in <module>
    NoneToken = Token("LOAD_CONST", offset=-1, attr=None, pattr=None)
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\uncompyle6\scanners\tok.py", line 59, in __init__
    from xdis.std import _std_api
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\xdis\std.py", line 214, in <module>
    _std_api = make_std_api()
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\xdis\std.py", line 212, in make_std_api
    return _StdApi(python_version, variant)
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\xdis\std.py", line 72, in __init__
    self.opc = opc = get_opcode_module(python_version, variant)
  File "c:\users\laike9m\appdata\local\programs\python\python37\lib\site-packages\xdis\op_imports.py", line 156, in get_opcode_module
    return op_imports[canonic_python_version[vers_str]]
KeyError: '3.7.5'
```

**Comment by laike9m** [comment]
Is it as easy as adding `3.7.5` [here](https://github.com/rocky/python-xdis/blob/master/xdis/magics.py#L330)?

**Comment by rocky** [comment]
Yes it is. Fixed in commit d80eb9e.  (3.7.5 and 3.8.0 were released recently.)

**Comment by laike9m** [comment]
Thanks.
